volatility3/framework/plugins/mac/__init__.py

```diff
@@ -0,0 +1,8 @@
+# This file is Copyright 2022 Volatility Foundation and licensed under the Volatility Software License 1.0
```

Comment:
The `__init__.py` file _shouldn't_ be necessary, because we import all python files under the various plugins directories, but I'm happy to have it for symmetry with the other plugins directories.  Thanks!  5:D